pandas/tests/arrays/test_integer.py

```diff
@@ -285,6 +294,21 @@ def test_error(self, data, all_arithmetic_operators):
         with pytest.raises(NotImplementedError):
             opa(np.arange(len(s)).reshape(-1, len(s)))
 
+    def test_pow(self):
+        a = pd.core.arrays.integer_array([1, None, None, 1])
+        b = pd.core.arrays.integer_array([1, None, 1, None])
+        result = a ** b
```

Comment:
note I don't think we have a test that uses None / np.nan in integer_array construction (e.g. that they are both actually interpreted the same) if you want to add one (this test of course implicitiy asserts this)

Reply:
Mostly works, just a failing case on `integer_array([None])` in https://github.com/pandas-dev/pandas/issues/23224